README.md

```diff
@@ -59,15 +59,15 @@ cd saleor-dashboard
 3. Install the dependencies:
 
 ```bash
-npm i
+pnpm install
```

Comment:
Something like this instead? Not all users likely know how to install pnpm:

```suggestion
npm install -g pnpm
pnpm install
```

Reply:
Added it as prerequisite